integration-cli/docker_api_containers_test.go

```diff
@@ -681,6 +681,48 @@ func (s *DockerSuite) TestContainerApiCreate(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestContainerApiCreateWithHostName(c *check.C) {
+	defer deleteAllContainers()
```

Comment:
seems we don't https://github.com/rajdeepd/docker/blob/dry-run-test/integration-cli/check_test.go#L30